deephaven/vscode-deephaven
# feat: Panels associated by id with specific connections

Replaced panel registry with new PanelService
- Panels are now associated with connections by variable id (they were mapped by title before across all connections)
- Should be able to re-use the same variable name in different connections

**Testing**
- Start 2 servers and create a connection to both of them via the status bar connection picker (you also have the option of starting 2 pip servers which should auto create connections). This requires a venv setup with deephaven-server installed
- Run a script that produces a panel on the first connection
- Drag the script to the other connection and run it there
- Should see 2 panels with same title

resolves #90
